cogs/send_get_roles_reminders.py

```diff
@@ -123,11 +123,8 @@ async def send_get_roles_reminders(self) -> None:
             if not member_requires_opt_in_roles_reminder:
                 continue
 
-            hashed_member_id: str = SentGetRolesReminderMember.hash_member_id(member.id)
             sent_get_roles_reminder_member_exists: bool = (
-                await SentGetRolesReminderMember.objects.filter(
-                    hashed_member_id=hashed_member_id,
-                ).aexists()
+                await SentGetRolesReminderMember.objects.filter(discord_id=member.id).aexists()  # type: ignore[misc]
```

Comment:
![image](https://github.com/CSSUoB/TeX-Bot-Py-V2/assets/18513864/a6e4bbf9-0edc-4254-8762-96290bcf6893)

ruh roh

Reply:
ohhhhh, i know how to solve this.